pkg/mysqlsh/mysqlshexec.go

```diff
@@ -33,7 +33,7 @@ func (m *mysqlshExec) runWithExec(ctx context.Context, cmd string) error {
 	if err != nil {
 		sout := sensitiveRegexp.ReplaceAllString(outb.String(), ":*****@")
 		serr := sensitiveRegexp.ReplaceAllString(errb.String(), ":*****@")
```

Comment:
Can we add the same for cmd?

Reply:
Yes we can, but I feel it is unnecessary and just bloats the log. From the stack you can clearly see where the error happened and check it.